src/Timestamps/MediumTimestampsHTMLFormatter.php

```diff
@@ -16,10 +16,10 @@ class MediumTimestampsHTMLFormatter implements TimestampsFormatterInterface
 
     private $fmtDay;
 
-    public function __construct()
+    public function __construct($locale)
```

Comment:
Old default value is missing here, this would be a BC break causing issues for existing uses of the library.